packages/paste-core/primitives/box/__tests__/box.test.tsx

```diff
@@ -9,27 +9,27 @@ import {CustomizableBoxExample} from '../__fixtures__/CustomizableBox';
 describe('Backgrounds', () => {
   it('should render single values', (): void => {
     render(
-      <Theme.Provider theme="console">
+      <Theme.Provider theme="default">
         <Box backgroundColor="colorBackgroundPrimary" data-testid="box">
           background single
         </Box>
       </Theme.Provider>
     );
     const renderedBox = screen.getByTestId('box');
-    expect(renderedBox).toHaveStyleRule('background-color', 'rgb(2,122,197)');
+    expect(renderedBox).toHaveStyleRule('background-color', 'rgb(2,99,224)');
```

Comment:
Again, here I think we should switch to the customization provider and the global mock   theme.

The tests are valid, we're asserting that token name equals a css property value, but we should just use a theme fixture to prevent test failures when we change token values.

Reply:
Switched to Customization provider.